packages/wallets/bitkeep/src/adapter.ts

```diff
@@ -181,3 +360,4 @@ export class BitKeepWalletAdapter extends BaseMessageSignerWalletAdapter {
         }
     }
 }
+
```

Comment:
All of this code should be simplified -- these shouldn't be literal copies of the code, they should just be exported variables and types that alias the names of the `Bitget...` ones so they don't break the exports.